public/css/app.css

```diff
@@ -23,6 +25,24 @@ body {
   display: grid;
 }
 
+a:visited {
+  color: var(--light);
+}
+
+a {
+  text-decoration: none;
+}
+
+.mininav {
+  display: flex;
+  justify-content: space-around;
+}
+
+.link-item {
+  height: 30px;
+  width: 100%;
+}
+
 /* .page-content {
     background-color: var(--darkest);
     margin: 0;
```

Comment:
Code that is commented out should rarely ever be checked into version control. It's better to delete the code. If you do need it later on you can find it in your previous commits.